### PR TITLE
Feature/607

### PR DESCRIPTION
**Pull request by szafran89**
Issue #607

### CODE REVIEW

**Comment by pkarw** [comment]
Thanks, good job. Waiting for @filrak to review and merge it in :)

File: doc/plugins/Working with plugins.md [hunk 1]
```diff
@@ -1,7 +1,18 @@
 # Working with plugins
 
-# Core plugins
+Plugins usually add global-level functionality to Vue. In Vue Storefront there are two types of plugins.
+
+## Core plugins
+
+Core plugins are located in `src/components` folder. There are two core plugins:
```
Comment (filrak) [inline doc/plugins/Working with plugins.md]: Small mistake. You mean they are in `src/plugins`?

File: doc/plugins/Working with plugins.md [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+
+* Config - it allows global access to config file (available via `$this.$config`),
+* Event Bus - it allows global event bus for the project (available via `$this.$bus`).
+
+## Theme plugins
+
+Theme plugins should be located in `src/themes/{theme_name}/plugins`. To add new plugin you have to keep the same structure as in core plugins.
```
Comment (filrak) [inline doc/plugins/Working with plugins.md]: Maybe change this 2 sentences to 'Theme plugins should be located in `src/themes/{theme_name}/plugins` in order to work'. It's more understandable. There should be also information where and how to register core and theme plugins.

File: doc/plugins/Working with plugins.md (hunk 1 above)
Comment (szafran89) [reply doc/plugins/Working with plugins.md]: Fixed

File: doc/plugins/Working with plugins.md (hunk 2 above)
Comment (szafran89) [reply doc/plugins/Working with plugins.md]: Fixed

**Pull request merged**